R/EMODnetWFS-package.R

```diff
@@ -7,5 +7,12 @@
 NULL
 
 emodnetwfs_user_agent <- function() {
+  if (whoami::gh_username() %in% c("annakrystalli", "salvafern", "maelle")) {
```

Comment:
Is this hardwiring the best approach here do you think longer term? I'm wondering whether using the github API to either get a list of collaborators or the members of a team that manages the package instead?

Reply:
I see your point but this would add a fragility?

Reply:
How do you mean fragility? I guess if we stuck to this approach, we should add some contributing guidelines and a note to add developer user names before contributing (for active maintainers-developers). It might be best to put the developers names in a config file in inst which is loaded by the function so that the R source code doesn't need to be changed everytime the Dev team changes?

Reply:
awesome idea!